chromium_src/third_party/blink/renderer/modules/storage/dom_window_storage.cc

```diff
@@ -82,7 +82,8 @@ EphemeralStorageNamespaces::EphemeralStorageNamespaces(
     StorageController* controller,
     const String& session_storage_id,
     const String& local_storage_id)
-    : session_storage_(
+    : Supplement(nullptr),
```

Comment:
this doesn't seem right, why would it be nullptr?

Reply:
This just keeps parity with the current implementation: we weren't initializing Supplement base class here, but now Supplement class c'tor requires an explicit argument (https://chromium.googlesource.com/chromium/src/+/75b2e73):
<pre>
Existing uses of the default ctor were replaced with:
- Supplement(T&), in cases where there was a Supplementable passed to
  the ctor.
- Supplement(nullptr), in cases where there was no Supplementable
  passed.
</pre>

Upstream change:
<pre>
-  Supplement() {}
+  explicit Supplement(std::nullptr_t) {}
</pre>